api/v1alpha1/etcdpeer_types.go

```diff
@@ -7,10 +7,26 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// BootstrapPeer describes a single *other* peer that will be used for bootstrapping
```

Comment:
Perhaps a `describes a single peer's connection details within a cluster`? This doesn't necessarily refer to only *other* peers ('itself' will also be included in this list), and clarifying `connection details` makes it clearer what this is used for